### PR TITLE
2 task

### PR DESCRIPTION
**Pull request by Levii01**
Add column admin to users table and write a very simple spec.
Update README with actual tasks.

### CODE REVIEW

**Comment by socjopata** [comment]
I'll wait with the review as the PR is wrong. If you create a feature branch off another feature branch, you have to create a PR accordingly (don't point to master). Otherwise I see non-relevant commits/code.

**Comment by Levii01** [comment]
@socjopata  ready :+1:

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -22,5 +22,9 @@
 
       it { is_expected.to_not be_valid }
     end
+
+    describe 'database columns' do
```
Comment (socjopata) [inline spec/models/user_spec.rb]: Specs are usually a nice thing, but in this specific case I think we are not testing our functionality, but how rails/migration work. And these are tested by default, with the specs written for the framework we are using ;)

**Review by socjopata — APPROVED** [review]
Consider my comment about specs, and then :shipit:

File: spec/models/user_spec.rb (hunk 1 above)
Comment (Levii01) [reply spec/models/user_spec.rb]: Thanks, I didn't know about it. It will be better if I delete spec for db column.

**Pull request merged**